nodejs/FatBeaconPeripheral/FatBeacon.js

```diff
@@ -0,0 +1,60 @@
+var bleno = require('bleno');
+var eddystoneBeacon = require('eddystone-beacon');
+
+/*************************Altering Library Functions****************************/
+
+var AdvertisementData = require('eddystone-beacon/lib/util/advertisement-data');
+var Eir = require('eddystone-beacon/lib/util/eir');
+var FAT_BEACON_FRAME_TYPE = 0x0e;
+
+var makeUrlBufferReplacement = function(name) {
+  console.log('AdvertisementData.makeUrlBuffer Called with ' + name);
+
+  var FAT_BEACON_FRAME_TYPE = 0x0e;
+  var MAX_URL_LENGTH = 18;
+  var SERVICE_UUID = 'feaa';
+  
+  var encodedName = Buffer.from(name);
+
+  if (encodedName.length > MAX_URL_LENGTH) {
+    throw new Error('Encoded Name must be less than ' + MAX_URL_LENGTH +
```

Comment:
You may want to try using template literals. (I think that's the way to do things these days)